source/includes/fact-mmapv1-big-endian.rst

```diff
@@ -1,5 +1,7 @@
 .. important::
 
-   MMAPv1 is not supported on big-endian architectures such as s390x.
-   MongoDB returns an error if you set MMAPv1 as the
-   storage engine on a big-endian system.
+   MMAPv1 is unsupported on big-endian architectures like the IBM
+   z/Architecture (``s390x``) and certain bi-endian architectures like
+   the PowerPC (``ppc64le``) platform. MongoDB returns an error if you
+   set ``MMAPv1`` as the storage engine for either the ``s390x`` or
+   ``ppc64le`` platforms. MMAPv1 was deprecated in MongoDB 4.0.
```

Comment:
I realised that this include is also brought in to /core/mmapv1. Having this note here is kind of redundant as that page has a big deprecation notice at the top. I think it's OK to remove it. There shouldn't be any surprises w.r.t. MMAPv1 at this point.

Reply:
Removed.